### ISSUE TITLE
training program idle

### ISSUE THREAD

**Issue by Cli98** [open-mmlab/mmdetection#2123]
When I use MMdetection to train FasterRCNN FPN, the program idle when training(no log on console for 9 hours). There is no loss nor new checkpoint generated. Even no error reported. I use 4 gpu to train the model with batch_size of 1. Disk space is available at the time error shows. No OOM error at the time I interrupt program.

Check following capture.
<img width="529" alt="image" src="https://user-images.githubusercontent.com/19630616/74953880-40948680-53d0-11ea-94b9-0e9b0bd3f6e1.png">

And capture of Memory usage at the time of interruption.(Observed issues at around 10 a.m.)
<img width="535" alt="image" src="https://user-images.githubusercontent.com/19630616/74953949-586c0a80-53d0-11ea-92f9-539f4651833b.png">

This happens lots of time When I train with backbone of ResNet 101Xt. I believe this should be a bug. @hellock Can you guys investigate?

**Comment by hellock** [comment]
Such phenomenon is occasionally observed when using pytorch, not only in mmdetection. It may be related to lots of issues (training codes, hardwares, cuda, cudnn, etc) and is not easy to debug.
As a quick solution, I suggest stopping the training when you find it stuck and then resume from the latest checkpoint. If you can reproduce the issue with certainty, we can further investigate it.

**Comment by Cli98** [comment]
Thanks for your suggestion. I'll check my side to see if I can figure out something.
